apps/datahub/src/app/record/record-apis/record-apis.component.html

```diff
@@ -21,14 +31,25 @@
     [link]="link"
     [currentLink]="selectedApiLink"
     size="S"
-    [ngClass]="{
-      'card-shadow': link !== selectedApiLink || !selectedApiLink,
-      'bg-neutral-100': link === selectedApiLink,
-    }"
     (openRecordApiForm)="openRecordApiForm($event)"
   >
   </gn-ui-api-card>
 </gn-ui-carousel>
+<gn-ui-block-list
+  #blockList
+  class="hidden md:block"
+  containerClass="gap-4 pt-5 pb-7"
+>
+  <gn-ui-api-card
```

Comment:
so you are duplicating elements to handle card sized? couldn't this have been avoided? this probably would add some more work, so, ok to keep like this.